.travis.yml

```diff
@@ -1,20 +1,29 @@
 env:
   global:
-    - KUBE_VERSION: 1.16
+    - KUBE_VERSION: 1.20
+    - K8S_RELEASE: 1.20
+
 language: go
 go:
-  - 1.12.1
+  - 1.15.5
 
 install:
 - export PATH=$GOPATH/bin:$PATH
 - mkdir -p $GOPATH/src/k8s.io/kubernetes
 - wget https://dl.k8s.io/v${KUBE_VERSION}.0/kubernetes-src.tar.gz -P $GOPATH/src/k8s.io/kubernetes
 - pushd $GOPATH/src/k8s.io/kubernetes && tar xzf kubernetes-src.tar.gz && popd
-- pushd $GOPATH/src/k8s.io/kubernetes && make generated_files && cp -L -R vendor ${GOPATH}/src/ && rm -r vendor && popd
+- pushd $GOPATH/src/k8s.io/kubernetes && make generated_files && cp -L -R vendor ${GOPATH}/src/ && popd
+- pushd $GOPATH/src/k8s.io/kubernetes && rm -r vendor/github.com/spf13/cobra && rm -r vendor/github.com/spf13/pflag && popd
+- pushd $GOPATH/src/k8s.io/kubernetes && cp -R vendor/k8s.io/kubectl/pkg/cmd  pkg/kubectl/ && popd
```

Comment:
no. this hack is too dirty.
We may want to change the import from k8s.io/kubectl/pkg/cmd directly, rather than from k8s.io/kubernetes/pkg/kubectl/cmd.

Reply:
Yes, agreed.
The source for the reference doc generators should be built and tested before merging.
The library problems/vendor hacking was a problem with the last several releases. It is an issue.
I think it is worthwhile to change/update the build tool and improve the build/test processes.